### PR TITLE
Currency:Add build script for configbump: v0.1.5

### PR DESCRIPTION
**Pull request by shreyakajbaje**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Comment by shreyakajbaje** [comment]
attaching logs for reference - [configbumpBuildTestsLogs.txt](https://github.com/ppc64le/build-scripts/files/10932195/configbumpBuildTestsLogs.txt)

**Comment by shreyakajbaje** [comment]
Additional Details - 

Machine: Power Linux 03af3a271a93 3.10.0-1160.6.1.el7.ppc64le
OS NAME="Red Hat Enterprise Linux"
OS VERSION="8.7 (Ootpa)"
GO VERSION: 1.13

File: c/Configbump/build-info.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "package_name" : "configbump",
+    "github_url":"https://github.com/che-incubator/configbump.git",
+    "version": "v0.1.5",
+    "default_branch": "main",
+    "package_dir": "c/Configbump",
```
Comment (nikhil-kalbande) [inline c/Configbump/build-info.json:6]: @shreyakajbaje we expect the package folder will all small case, so please update directory name. `c/configbump`

File: c/Configbump/configbump_ubi_8.5.sh [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+GO_VERSION=${GO_VERSION:-1.13}
+
+wget https://golang.org/dl/go$GO_VERSION.linux-ppc64le.tar.gz
+tar -C /bin -xf go$GO_VERSION.linux-ppc64le.tar.gz
+
+rm -f go$GO_VERSION.linux-ppc64le.tar.gz
+
+export PATH=$PATH:/bin/go/bin
+export GOPATH=/home/go
```
Comment (nikhil-kalbande) [inline c/Configbump/configbump_ubi_8.5.sh:35]: @shreyakajbaje This doesn't look correct and may break in different environment.
Refer - https://github.com/ppc64le/build-scripts/blob/master/o/opentelemetry-go-contrib/opentelemetry-go-contrib_ubi_8.5.sh#L26-L29

**Review by nikhil-kalbande — CHANGES_REQUESTED** [review]

**Review by nikhil-kalbande — APPROVED** [review]

**Pull request merged**